applications/nrf5340_audio/src/audio/sw_codec_select.h

```diff
@@ -52,19 +57,25 @@ struct sw_codec_encoder {
 
 struct sw_codec_decoder {
 	bool enabled;
+	uint8_t audio_mode;
 	uint8_t num_ch;
 	enum audio_channel audio_ch; /* Only used if channel mode is mono */
```

Comment:
audio_mode, num_ch and audio_ch are very similar, we could improve this, or adding comment to explain the details in here.

Reply:
Yes, I will probably rename this at some point as Kristoffer also thought it was a bit vague. Hopefully this will make it more clear